src/clj/game/cards/programs.clj

```diff
@@ -2785,6 +2785,55 @@
                                                  (trigger-event state :corp :no-action))}])
                                 (make-run eid :archives (get-card state card)))}]})
 
+(defcard "Sneakdoor Prime A"
+         {:abilities [{:cost [:click 2]
+                       :prompt "Choose a server"
+                       :choices (req (cancellable
+                                       (->> runnable-servers
+                                            (map unknown->kw)
+                                            (filter is-remote?)
+                                            (map remote->name))))
+                       :msg "make a run on a remote server"
+                       :makes-run true
+                       :async true
+                       :effect (effect (register-events
+                                         card
+                                         [{:event :pre-successful-run
+                                           :duration :end-of-run
+                                           :unregister-once-resolved true
+                                           :prompt "Choose a server"
+                                           :choices (req (->> servers
+                                                              (map unknown->kw)
+                                                              (filter is-central?)
+                                                              (map central->name)))
```

Comment:
This can just be `["Archives" "R&D" "HQ"]`.

Reply:
👍🏻